src/rpc/rawtransaction_util.cpp

```diff
@@ -21,24 +21,15 @@
 #include <util/strencodings.h>
 #include <util/translation.h>
 
-CMutableTransaction ConstructTransaction(const UniValue& inputs_in, const UniValue& outputs_in, const UniValue& locktime, std::optional<bool> rbf)
+void AddInputs(CMutableTransaction& rawTx, const UniValue& inputs_in, const UniValue& locktime, std::optional<bool> rbf)
```

Comment:
In 41464504b74690dac173f0615f8fb2b444d8f0e9 "wallet: add `outputs` arguments to `bumpfee` and `psbtbumpfee`"

I don't think it makes sense to take `locktime` as a parameter to `AddInputs`. While the locktime can be relevant to the transaction's inputs, it doesn't make sense to be parsing and setting that value when adding the inputs. It would be better to have `ConstructTransaction` parse and set the locktime, and then `AddInputs` can just look at `rawTx.nLockTime` when it sets the inputs.